VideoApp/VideoApp/Stores/AppSettings/SubscribedTrackSwitchOffMode.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Copyright (C) 2021 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+enum SubscribedTrackSwitchOffMode: String, SettingOptions {
+    case sdkDefault
+    case auto
+    case manual
+    
+    var title: String {
+        switch self {
+        case .sdkDefault: return "SDK Default (Auto)"
```

Comment:
For `Server Default` we don't specify what the default actually is so maybe we shouldn't for SDK either. Perhaps unlikely to cause a problem but it is out of the app's control in any case.

Reply:
Updated.